recipes/libvpx/all/conanfile.py

```diff
@@ -0,0 +1,151 @@
+                              'tag_content WholeProgramOptimization true',
+                              'tag_content WholeProgramOptimization false')
+        msvs_common_sh = os.path.join(self._source_subfolder, "build", "make", "msvs_common.sh")
+        tools.replace_in_file(msvs_common_sh,
+                              'if [ "$(uname -o 2>/dev/null)" = "Cygwin" ] \\',
+                              'if [ "$(uname -o 2>/dev/null)" = "Cygwin" ] || [ "$(uname -o 2>/dev/null)" = "Msys" ]\\')
+
+    def _configure_autotools(self):
+        win_bash = tools.os_info.is_windows
+        prefix = os.path.abspath(self.package_folder)
+        if win_bash:
+            prefix = tools.unix_path(prefix)
+        args = ['--prefix=%s' % prefix,
+                '--disable-examples',
+                '--disable-unit-tests',
+                '--disable-tools',
+                '--enable-vp9-highbitdepth',
+                '--as=yasm']
+        if self.options.shared:
+            args.extend(['--disable-static', '--enable-shared'])
+        else:
+            args.extend(['--disable-shared', '--enable-static'])
+        if self.settings.os != 'Windows' and self.options.fPIC:
+            args.append('--enable-pic')
+        if self.settings.build_type == "Debug":
+            args.append('--enable-debug')
+        if self.settings.compiler == 'Visual Studio':
+            if 'MT' in str(self.settings.compiler.runtime):
+                args.append('--enable-static-msvcrt')
+
+        arch = {'x86': 'x86',
+                'x86_64': 'x86_64',
+                'armv7': 'armv7',
+                'armv8': 'arm64',
+                'mips': 'mips32',
+                'mips64': 'mips64',
+                'sparc': 'sparc'}.get(str(self.settings.arch))
+        build_compiler = str(self.settings.compiler)
+        if build_compiler == 'Visual Studio':
+            compiler = 'vs' + str(self.settings.compiler.version)
+        elif build_compiler in ['gcc', 'clang', 'apple-clang']:
+            compiler = 'gcc'
+        else:
+            raise ConanInvalidConfiguration("Unsupported compiler '{}'.".format(build_compiler))
+
+        build_os = str(self.settings.os)
+        if build_os == 'Windows':
+            os_name = 'win32' if self.settings.arch == 'x86' else 'win64'
+        elif build_os in ['Macos', 'iOS', 'watchOS', 'tvOS']:
+            os_name = 'darwin11'
+        elif build_os == 'Linux':
+            os_name = 'linux'
+        elif build_os == 'Solaris':
+            os_name = 'solaris'
+        elif build_os == 'Android':
+            os_name = 'android'
+        target = "%s-%s-%s" % (arch, os_name, compiler)
+        if tools.cross_building(self) or self.settings.compiler == 'Visual Studio':
+            args.append('--target=%s' % target)
+        if self.settings.compiler == 'apple-clang':
+            if float(str(self.settings.compiler.version)) < 8.0:
+                args.append('--disable-avx512')
```

Comment:
Let's move avx* to a disabled-by-default 'avx' option like in other codecs. Otherwise, it can cause non-portable binaries. SSE* can be enabled everywhere.

Reply:
sure

Reply:
added